homebrew/using_different_homebrew_formula_versions.md

```diff
@@ -0,0 +1,32 @@
+# Using different Homebrew formula versions
+
+Today, I was trying to run the executable from a gem and got the following error:
+
+```bash
+/.rbenv/versions/2.1.3/lib/ruby/gems/2.1.0/gems/rmagick-2.13.2/lib/RMagick.rb:11
+in `require': This installation of RMagick was configured with ImageMagick 6.8.9
+but ImageMagick 6.9.0-3 is in use. (RuntimeError)
```

Comment:
My previous comment got hidden when you pushed. Thoughts on only showing the last part of the error message?

```
This installation of RMagick was configured with ImageMagick 6.8.9
but ImageMagick 6.9.0-3 is in use. (RuntimeError)
```